docs/guides/text-tracks.md

```diff
@@ -225,6 +225,14 @@ As mentioned [above](#a-note-on-remote-text-tracks), remote text tracks represen
 * `Player#addRemoteTextTrack(Object options)`
 
   Available options are the same as the [available `track` attributes](#track-attributes). And `language` is a supported option as an alias for the `srclang` attribute - either works here.
+  
+  **Note**: If you need a callback, instead of a callback you could use the technique below:
+  ```js
+  const trackEl = player.addRemoteTextTrack({src: 'en.vtt'}, false);
+  trackEl.on('load', function() {
```

Comment:
we should use `addEventListener` because if native tracks are used `trackEl` will reference the native object.